src/main/java/pt/dinis/temporary/Configurations.java

```diff
@@ -0,0 +1,55 @@
+package pt.dinis.temporary;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+import org.apache.log4j.Logger;
+
+/**
+ * Created by diogo on 03-02-2017.
+ */
+public class Configurations {
+    private static final Logger logger = Logger.getLogger(Configurations.class);
+
+    private static Properties defaultProps, userProps;
+    private static final String propsFileName = "config.properties";
```

Comment:
http://stackoverflow.com/questions/7259687/java-naming-convention-for-static-final-variables
PROPERTIES_FILE_NAME
(properties em vez de props é a minha sugestão)